packages/steps/src/BuildStepInput.ts

```diff
@@ -227,6 +212,10 @@ export class BuildStepInput<
     }
   }
 
+  private parseInputValueToString(value: string): string {
+    return JSON.parse(`"${value}"`);
+  }
```

Comment:
What will happen if one passes string `test\ntest2` as input? I believe it will fail, right?

Can we guarantee that we will always use `\\n` instead of `\n` in inputs 🤔?

Reply:
I'm pretty sure it's guaranteed and all inputs are JSON.stringified.
If we want a fallback, I can bring back the SyntaxError check, in which case it will return the raw string